CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing to katello-client-bootstrap
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to **katello-client-bootstrap** and which is hosted on Github in the [Katello](https://github.com/Katello) Organization.
+These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for katello-client-bootstrap. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
+
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub issues](https://guides.github.com/features/issues/).
```

Comment:
Shouldn't that be a link to https://github.com/Katello/katello-client-bootstrap/issues?